frontend/pages/index.tsx

```diff
@@ -440,11 +440,11 @@ const Splash = (props: SplashProps): ReactElement => {
               ]}
             />
           )}
-          <SearchBarCheckboxItem
+          {SITE_ID !== 'fyh'&&<SearchBarCheckboxItem
```

Comment:
Use the `isClubFieldShown('target_schools')` function! It was designed for this purpose.